.circleci/config.yml

```diff
@@ -1,29 +1,54 @@
 workflows:
   version: 2
   circleci_tests:
+    # The following jobs runs in the following stages:
```

Comment:
Please add link to image (see other comment).

Reply:
done!